explorer-api/src/state.rs

```diff
@@ -30,15 +32,15 @@ impl ExplorerApiState {
             .get()
             .await
             .value
-            .iter()
-            .find(|node| node.mix_node.identity_key == pubkey)
-            .cloned()
+            .get(pubkey)
+            .map(|bond| bond.bond.clone())
```

Comment:
Maybe the argument should be something like `bond_with_location`, to avoid the naming collision.

Reply:
Renamed to `cache_item`